app/controllers/admin/users_controller.rb

```diff
@@ -119,37 +119,26 @@ def penalty_history
   end
 
   def suspend
-    SuspendUser.call(user: @user) do
+    User::Suspend.call do
```

Comment:
It’s best to not inject models as dependencies, a service should be able to fetch what it needs at that level (using the `model` step). I know `@user` is already being fetched by the controller, but long-term the idea would probably be to migrate the various controller’s actions, so the new services should not rely on it too much.

Regarding the renaming, it’s best to namespace services by concept when possible. It’ll help a bit organizing things when we’ll get hundreds of services :slightly_smiling_face: